Add init docs for OpenLab (#1)
theopenlab/publications: README.md

---

# Introduction
## OpenLab

OpenLab is a community-led program to test and improve support for the cloud-related SDKs/Tools as well as platforms like Kubernetes, Terraform, CloudFoundry and more. The goal is to improve the usability, reliability and resiliency of tools and applications for hybrid and multi-cloud environments.

> 

Reviewer: I think if we can add a step drawing at here to describe the order of `To be an OpenLab member`, `Test request`, `Development` and `Resource sponsoring`, it is better.

### Test request
If you have an idea or real use case about some integration scenarios like integrating Kubernetes and OpenStack, and plan to do proof of concept(POC) against some technologies, all you have to do is just sharing your idea and use case in OpenLab, if others are interested in your case, they will pick up your task and complete the subsequent testing work, pay attention on updating of OpenLab [web site](https://openlabtesting.org/explore/), we will share the test result on it. Start your test request [here](./contributor/Testing_Request.md) now.

### Development
The main work of OpenLab is to provide an auto integration testing for different upstream tool chains of OpenStack, e.g. Gophercloud, Terraform. Always we have to add CI jobs for the test requests submitted to OpenLab, if you are interesting in this job, please follow [development guide](contributor/Development_guide.md).

### Resource sponsoring

> The OpenLab has multiple resource pools(virtual machine, physical machine and device) provided by multiple cloud providers to run testing jobs, like VEXXHOST, OpenTelekomCloud, Citynetwork and so on. Thanks to OpenLab sponsors we also can offer the dedicated resources to users for their own tests. We'd very appreciate it if you would like to [donate](./contributor/Resource_Sponsor.md) resources to OpenLab.

Reviewer: ```suggestion
The OpenLab has multiple resource pools(virtual machine, physical machine and device) provided by multiple cloud providers to run testing jobs, like VEXXHOST, OpenTelekomCloud, Citynetwork and so on. Thanks to OpenLab sponsors we can offer both virtual machine and physical machine to users for their own tests. We'd very appreciate it if you would like to [donate](./contributor/Resource_Sponsor.md) resources to OpenLab.
```

---

# Introduction
## OpenLab

OpenLab is a community-led program to test and improve support for the most popular Software Development Kits(SDKs) as well as platforms like Kubernetes, Terraform, CloudFoundry and more. The goal is to improve the usability, reliability and resiliency of tools and applications for hybrid and multi-cloud environments.

## How to Join OpenLab

> There is plenty of room for helping the shape of future of OpenLab and ensuring things run smoothly. Welcome to join OpenLab if you're interested in our system, please follow the guide below:

Reviewer 2: ```suggestion
There is plenty of room for helping to shape of future of OpenLab and ensuring things run smoothly. If you are interested in our system, please follow the guidelines below:
```

### To be an OpenLab member
Join us to be a member of OpenLab firstly, then you will find out what's happening in OpenLab, what fellow open source community members are doing using OpenLab resources, also as a member you can access OpenLab resources for your testing, please follow the simple [member application](Member_Application.md) guide.

### Test Request

> If you wish to use OpenLab resources, start your test request [here](./contributor/Testing_Request.md).

Reviewer: If you have an idea or real use case about some integration scenarios, like: integrating Kubernetes and OpenStack, plan to do proof of concept(POC) against some technologies, all you have to do is just sharing your idea and case in OpenLab, if others are interested on your case, they maybe pick up your task and complete the subsequent testing works, pay attention on updating of OpenLab [web site](https://openlabtesting.org/explore/), we will share the test result on it. Start your test request [here](./contributor/Testing_Request.md) now.

### Development

> The main work of OpenLab CI is to provide an auto integration testing for different upstream tool chains of OpenStack, e.g. Gophercloud, Terraform. Always we have to add CI jobs for the test requests submitted to OpenLab, if you are interesting in this job, please follow [Development Guide](contributor/Development_Workflow.md).

Reviewer: s/[Development Guide]/[development guide]/

### Resource sponsoring

> The OpenLab has multiple resource pools provided by multiple cloud providers to run testing jobs. Thanks to our sponsors we also can offer the dedicated resources to users for their own tests. We'd very appreciate it if you would like to [donate](./contributor/Resource_Sponsor.md) resources to us.

Reviewer: 1. multiple resource pools(virtual machine, physical machine and device)
2. run testing jobs, like: VEXXHOST, OpenTelekomCloud, Citynetwork and so on.
3. resources to OpenLab.

> ## Submitting Your Publication

Reviewer: I prefer to remove this whole section `Submitting Your Publication`, this part content have been included in `Test Request` section

* clone this document repository
* create a branch with your request ID \(issue number provided when you requested resources\)

## References

> * OpenLab Web Site is hosted on [OpenLab](https://openlabtesting.org/)

Reviewer: s/OpenLab Web Site/OpenLab web site/

> * Join our IRC Channel `#askopenlab` on freenode.net

Reviewer: s/Channel/channel/

Reviewer: Add a link at the end, not everyone is familiar how to connect to freenode [See](https://freenode.net/kb/answer/chat)

> * The OpenLab CI github APP is hosted on [github APP](https://github.com/apps/theopenlab-ci)

Reviewer: I think we should switch the order of `github APP` and `CI jobs' status` in the list, let newbie to get interesting content at first

* Source code of OpenLab CI jobs is hosted on [openlab-zuul-jobs](https://github.com/theopenlab/openlab-zuul-jobs)

> * CI Jobs' status: [Status](http://status.openlabtesting.org/t/openlab/status.html)

Reviewer: ```
OpenLab CI Jobs' [status](http://status.openlabtesting.org/t/openlab/status.html)
```
